lib/lynx/command/basic.rb

```diff
@@ -10,7 +10,7 @@ class << self
         def instruct(method, &block)
           define_method(method) do |*args|
             @command << block.call(self,*args)
-
+            print("...")
```

Comment:
What does this do?

Reply:
This is mostly just to add visual feedback that the commands are being executed without actually printing the output of the commands. I added this after noticing that upon executing bin/setup, it looks like the script hangs (when it really isn't). I'm sure we could come up with something that's a bit more informative (like abbreviated mysql commands) if we'd like

Reply:
This might be something we want to add to to `bin/setup` itself - instead of this library?